framework/source/class/qx/ui/core/scroll/AbstractScrollArea.js

```diff
@@ -267,9 +257,10 @@ qx.Class.define("qx.ui.core.scroll.AbstractScrollArea",
     */
 
     /**
-     * Returns the boundaries of the pane.
+     * Returns the dimensions of the pane.
      *
-     * @return {Map} The pane boundaries.
+     * @return {{width: number, height: number}|null} The pane dimension in pixel. Contains
```

Comment:
I just build with the API viewer. The latest changes also show the width and height nicely in the documentation page of the form.List. No problems in the API viewer anymore.